### DISCUSSION TITLE
Why my bot sell runes? And stop pick up EVERY items when get 500k Gold in stash?

### DISCUSSION THREAD

**Discussion by GogolPL** [blizzhackers/kolbot#423] [General]
Hello guys.
I just beggin game on d2 so i play with 1 charracter soso light.. 1st time use bot and try to do everything without help but this is weard problem...

Config use to play : D2BotLead.dbj (Its only one who work and move when i setting)
I have set kolton.nip like this:

//[name] == elrune # # [MaxQuantity] == 1
//[name] == eldrune # # [MaxQuantity] == 1
//[name] == tirrune # # [MaxQuantity] == 1
//[name] == nefrune
//[name] == ethrune # # [MaxQuantity] == 1
//[name] == ithrune # # [MaxQuantity] == 1
//[name] == talrune # # [MaxQuantity] == 1
//[name] == ralrune # # [MaxQuantity] == 1
//[name] == ortrune # # [MaxQuantity] == 2
//[name] == thulrune # # [MaxQuantity] == 2
//[name] == amnrune # # [MaxQuantity] == 1
//[name] == solrune # # [MaxQuantity] == 2
//[name] == shaelrune # # [MaxQuantity] == 1
//[name] == dolrune # # [MaxQuantity] == 1
//[name] == helrune # # [MaxQuantity] == 2
[name] == iorune # # [MaxQuantity] == 1
[name] == lumrune # # [MaxQuantity] == 1
[name] == korune # # [MaxQuantity] == 1
[name] == falrune
[name] == lemrune # # [MaxQuantity] == 1
[name] == pulrune # # [MaxQuantity] == 1
[name] == umrune # # [MaxQuantity] == 1
[name] == malrune # # [MaxQuantity] == 1

but my bot sell whole time runes to akara.. what i had to change??

And one more Question. My bot stop pick iteams when get 500k in stash and i have no IDEA why.. Can you help me please???

**Comment by theBGuy** [comment]
You didn't uncomment kolton.nip in your character config

**Reply by GogolPL** [reply]
I have right this:	

// ########################### //
	/* ##### PICKIT SETTINGS ##### */
	// ########################### //
	// Default folder is kolbot/pickit.
	// Item name and classids located in NTItemAlias.dbl or modules/sdk.js

	Config.PickitFiles.push("kolton.nip");
	Config.PickitFiles.push("LLD.nip");
	Config.PickRange = 40; // Pick radius
	Config.FastPick = false; // Check and pick items between attacks
	Config.OpenChests.Enabled = false; // Open chests. Controls key buying.
	Config.OpenChests.Range = 15; // radius to scan for chests while pathing
	Config.OpenChests.Types = ["chest", "chest3", "armorstand", "weaponrack"]; // which chests to open, use "all" to open all chests. See sdk/chests.txt for full list of chest names


Before was like this:

	// ########################### //
	/* ##### PICKIT SETTINGS ##### */
	// ########################### //
	// Default folder is kolbot/pickit.
	// Item name and classids located in NTItemAlias.dbl or modules/sdk.js

	**_//Config.PickitFiles.push("kolton.nip");
	//Config.PickitFiles.push("LLD.nip");_**
	Config.PickRange = 40; // Pick radius
	Config.FastPick = false; // Check and pick items between attacks
	Config.OpenChests.Enabled = false; // Open chests. Controls key buying.
	Config.OpenChests.Range = 15; // radius to scan for chests while pathing
	Config.OpenChests.Types = ["chest", "chest3", "armorstand", "weaponrack"]; // which chests to open, use "all" to open all chests. See sdk/chests.txt for full list of chest names

Now its ok????

**Comment by GogolPL** [comment]
Ok.... I go it it save items at chest, but now he didnt pickanything when got 100k in stash...
